quickannotator/api/v1/project/__init__.py

```diff
@@ -56,8 +56,7 @@ def post(self, args):
         """     create a new Project
         """
 
-        db.session.add(qadb.Project(name=args['name'], description=args['description'], is_dataset_large=args['is_dataset_large']))
-        db.session.commit()
+        db_session.add(models.Project(name=args['name'], description=args['description'], is_dataset_large=args['is_dataset_large']))
```

Comment:
is this large boolean coming from the UI where the user specifies project size? might be better to rename this from "large" to an explicit value that was used in the UI so that the code ism ore transparent? e.g., "is_dataset_larger50k" or something?

Reply:
This was kind of a loose placeholder because we didn't brainstorm functionality that would fundamentally change based on the project size. 

I think we discussed warning the user that large projects may require more compute, i.e., a larger ray cluster?

Reply:
yes, thats right, thanks for the reminder. generally i just take issue with the vagueness of "large", since your definition of large may not be the same for that of the user. this definitely means that we'll need to include clear instructions in the front end (obviously, and i believe thats noted in the figma as well), but i worry that e.g., a backend developer won't know what this means since they won't read the front end code. for stuff like that, which is ambigious in nature, its often customary to add a comment where the variable is defined so folks can "stumble" upon the relevant information without having to wonder. so maybe we just add a note in the model definition and call it a day?